rover/src/owr_drive_controls/src/fourWheelDrive.cpp

```diff
@@ -16,13 +16,19 @@
 #define FRONT 1
 #define BACK -FRONT
 
+#define FWDRIVE_LIMIT 0.75*M_PI/2
 
 motorVels FourWheelDrive::steer(motorVels vels, double velMagnitude, double turnAngle) {
     motorVels output = vels;
     output.frontLeftMotorV = velMagnitude;
     output.backLeftMotorV = velMagnitude;
     output.frontRightMotorV = velMagnitude;
     output.backRightMotorV = velMagnitude;
+
+    // Capping the turnangle to 75% of M_PI/2
+    if(turnAngle >= FWDRIVE_LIMIT) { turnAngle = FWDRIVE_LIMIT; }
+    else if(turnAngle <= -FWDRIVE_LIMIT) { turnAngle = -FWDRIVE_LIMIT; }
```

Comment:
this would be more readable if the code after the opening brackets was on a new line or it was in an inline function.

Reply:
Thanks, done